### PR TITLE
Only use the closure compiler's JSON.parse as a fallback...otherwise …

### PR DESCRIPTION
**Pull request by jsaleigh**
…'eval' is used instead by default, which can cause Content Security Policy issues.

### CODE REVIEW

**Comment by jsaleigh** [comment]
Now that I look at it, this could fail.  goog.json.parse has exception handling, so a straight replacement loses that while it gets rid of eval.

File: docs/web/3_branch_web.md [hunk 1]
```diff
@@ -464,7 +464,7 @@ branch.deepview(data, option, function(err) {
     if (err) {
         throw err;
     }
-    $('a.deepview-cta').click(branch.deepviewCta);
+    ${'a.deepview-cta').click(branch.deepviewCta);
```
Comment (chanian) [inline docs/web/3_branch_web.md]: Typo here?

**Comment by chanian** [comment]
:+1: contingent on fixing typo in docs

File: docs/web/3_branch_web.md (hunk 1 above)
Comment (jsaleigh) [inline docs/web/3_branch_web.md]: Wow where did that come from...

**Pull request merged**